classes/test.php

```diff
@@ -1826,6 +1830,14 @@ private static function cleanNamespace($namespace)
 
 	private static function isRegex($namespace)
 	{
-		return preg_match('/^([^\\\[:alnum:][:space:]]).*\1.*$/', $namespace) === 1;
+		return is_numeric(@preg_match($namespace, 'foo'));
```

Comment:
Instead of hardcoding a random value, I would have used `null` to replace `' foo'`